### DISCUSSION TITLE
I get stuck during the gaokao-gen evaluation

### DISCUSSION THREAD

**Discussion by liu904-61** [open-compass/opencompass#621] [Q&A]
![image](https://github.com/open-compass/opencompass/assets/99941000/673ecb93-fccc-4038-bb65-db4954eb9395)
I want to know why I'm staying here。

**Comment by tonysy** [comment]
Please check the `*.out` file in `outputs/` folder

**Reply by liu904-61** [reply]
Oh, sorry, it's just too slow, and I thought it got stuck.
So, how should I make it faster?

**Reply by tonysy** [reply]
Please provide more information about your model  and configuration.
Also, welcome to check this first: https://opencompass.readthedocs.io/en/latest/get_started/faq.html#efficiency

**Reply by liu904-61** [reply]
model：llama2-70b_hf，datasets：gaokao-gen ，server：8*A100，config：default
